### PR TITLE
LPS-148367 [CP 2.0] Assign incident contact roles when customer is filling out activation forms

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-148367](https://liferay.atlassian.net/browse/LPS-148367)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/components/IncidentContactEditModal/index.jsx [hunk 1]
```diff
@@ -51,19 +59,66 @@ const IncidentContactEditModal = ({
...
+
 			removeHighPriorityContactList?.map((item) => {
-				return removeHighPriorityContactsList(client, item);
-			})
-		);
+				openToast(
+					`${item.label}`,
+					`${i18n.translate('high-priority-contact-removed')} 
+						${i18n.translate(`${getKebabCase(item.filter.name)}-contact`)}`
+				);
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/components/IncidentContactEditModal/index.jsx:103]: I think it is better to capitalize the first character of the user name, do you agree?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/components/IncidentContactEditModal/index.jsx [hunk 2]
```diff
@@ -51,19 +59,66 @@ const IncidentContactEditModal = ({
...
 			addHighPriorityContactList?.map((item) => {
-				return addHighPriorityContactsList(client, item);
-			})
-		);
-
-		close();
+				openToast(
+					`${item.label}`,
+					`${i18n.translate('high-priority-contact-added')} 
+						${i18n.translate(`${getKebabCase(item.category.name)}-contact`)}`
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/components/IncidentContactEditModal/index.jsx:110]: Make the name of the high-priority category bold, please 🔨

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx [hunk 3]
```diff
@@ -34,6 +47,7 @@ const SetupHighPriorityContact = ({
 			contactsCategory: {
 				key: _filter,
 				name: `${filter}`,
+				role: getContactRoleByFilter(`${filter}`),
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx:50]: Do we really need template strings here?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 4]
```diff
@@ -183,6 +183,8 @@
 	"group-id-confirmation": "Group ID Confirmation",
 	"hamina-finland": "Hamina, Finland",
 	"hide": "Hide",
+	"high-priority-contact-removed": "successfully added from",
+	"high-priority-contact-added": "successfully removed from",
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json]: I think you've switched the translations here. Also, change the alphabetical order.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 5]
```diff
@@ -183,8 +183,8 @@
 	"group-id-confirmation": "Group ID Confirmation",
 	"hamina-finland": "Hamina, Finland",
 	"hide": "Hide",
-	"high-priority-contact-removed": "successfully added from",
 	"high-priority-contact-added": "successfully removed from",
+	"high-priority-contact-removed": "successfully added from",
```
Comment (pedroliveirav) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:187]: Same here, I think you've changed the translations here.

**Review by pedroliveirav — COMMENTED** [review]
Needs some fixes 🚀

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json (hunk 5 above)
Comment (eduardo-diniz) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:187]: DOne! thx

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx (hunk 3 above)
Comment (eduardo-diniz) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx:50]: thanks!

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Review by pedroliveirav — APPROVED** [review]
Good job! 🚀 🚀

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**